@xen-orchestra/proxy/src/app/index.js

```diff
@@ -39,5 +40,12 @@ export default class App {
     keys(descriptors).forEach(name => {
       noop(this[name])
     })
+
+    // dispose all created resources on stop
+    this.hooks.once('stop', () => debounceResource.flushAll().catch(() => {}))
```

Comment:
Use `promise-toolbox/ignoreErrors` instead to avoid catching programmer errors.

Reply:
After thought, it shouldn't be cached. The hook will properly handle and log the error.

Reply:
What's the behavior in case it not flushed?